// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

import "components/feed/core/proto/v2/wire/feed_request.proto";

option optimize_for = LITE_RUNTIME;

message Request {
  enum RequestVersion {
    UNKNOWN_REQUEST_VERSION = 0;
    FEED_QUERY = 1;
  }
  optional RequestVersion request_version = 1;
  optional FeedRequest feed_request = 1000;
}
